bundles/org.openhab.binding.awattar/src/main/java/org/openhab/binding/awattar/internal/AwattarBestPriceResult.java

```diff
@@ -47,7 +49,18 @@ public void updateEnd(long end) {
         }
     }
 
-    public abstract boolean isActive();
+    /**
+     * Returns true if the best price is active.
+     *
+     * @param now the current time
+     * @return true if the best price is active, false otherwise
+     */
+    public abstract boolean isActive(Instant now);
```

Comment:
See other comment, better to change tthe parameter name

Reply:
done